godot_project/autoloads/data/ElementData.gd

```diff
@@ -76,3 +76,9 @@ func _get_contact_radius() -> float:
 	var fallback_factor: float = ProjectSettings.get_setting(VAN_DER_WAALS_FALLBACK_FACTOR_SETTING, 1.5)
 	return render_radius * fallback_factor
 
+
+func get_maximum_bonds_count() -> int:
+	var valence_left: int = valence
+	if number > 5:
+		valence_left = 8 - valence_left
+	return valence_left
```

Comment:
this method is never used and is also wrong, as an example valence of helium is 2, but bonds count of helium is meant to be 0

Reply:
Right I forgot to remove that
But if this is wrong, then this one is wrong too I guess? https://github.com/MSEP-one/msep.one/blob/main/godot_project/project_workspace/structs/atomic_structure.gd#L343

Reply:
we could try to correct the hydrogens on helium and see what happens